web/src/list_widget.ts

```diff
@@ -3,6 +3,28 @@ import $ from "jquery";
 import * as blueslip from "./blueslip";
 import * as scroll_util from "./scroll_util";
 
+type ListWidget<Key = unknown, Item = Key> = {
+    get_current_list(): Item[];
+    filter_and_sort(): void;
+    retain_selected_items(): void;
+    render(how_many?: number): void;
+    render_item(item: Item): void;
+    clear(): void;
+    set_filter_value(value: string): void;
+    set_reverse_mode(reverse_mode: boolean): void;
+    set_sorting_function(sorting_function: string): void;
+    set_up_event_handlers(): void;
+    clear_event_handlers(): void;
+    increase_rendered_offset(): void;
+    reduce_rendered_offset(): void;
+    remove_rendered_row(row: HTMLElement): void;
```

Comment:
I think we are getting a JQuery element from the call site:
https://github.com/zulip/zulip/blob/27878cce87c973100979e8a6fd8b33705b934fee/web/src/recent_topics_ui.js#L489-L493
https://github.com/zulip/zulip/blob/27878cce87c973100979e8a6fd8b33705b934fee/web/src/recent_topics_ui.js#L586-L608

Reply:
`topic_row` is not consistent with our naming scheme for JQuery elements with a `$` prefix, so perhaps that can be fixed too.